sql/bench_test.go

```diff
@@ -29,6 +30,57 @@ import (
 	"github.com/cockroachdb/cockroach/server"
 )
 
+func benchmarkPGWire(b *testing.B, f func(b *testing.B, db *sql.DB)) {
```

Comment:
`s/PGWire/Cockroach/` in all benchmark names? `PGWire` made sense when we were comparing to RPC and HTTP, but less so in comparison to `Postgres` and `MySQL`. Maybe `CockroachPG` to leave open the possibility of other protocols in the future.